### ISSUE TITLE
Enhance testing for `validate_attestation_signature`

### ISSUE THREAD

**Issue by paulhauner** [sigp/lighthouse#94]
## Description

There are scenarios in `attestation_validation::validate_attestation_signature` that are untested.

You can find the crate at `/beacon_chain/attestation_validation` (once #85 has been merged into master).

Here is a non-comprehensive list to get you started:

- No signatures at all (aggregate signature is empty/at infinity).
- There is an extra signature on the attestation.
- The database fails.
- A validator index is unknown.

## Steps to resolve

1. Determine all code paths in the `validate_attestation_signature` function (within reason -- ask here for canonical definition of "reason")
2. Implement tests for reasonable code paths.

**Comment by ralexstokes** [comment]
what is this blocked on? just impl'ing the latest logic per the spec?

**Comment by paulhauner** [comment]
I don't believe it is blocked :)

**Comment by Feng94** [comment]
I would like to try this one if nobody's already working on it :)

**Comment by paulhauner** [comment]
Unfortunately this function has been removed with spec updates! Buzz me on Gitter if you're keen for something else, I'm going to make lots of new issues!
